kapps/Dockerfile

```diff
@@ -0,0 +1,64 @@
+ARG docker_repo=zokradonh
+FROM ${docker_repo}/kopano_base:latest
+
+ARG ADDITIONAL_KOPANO_PACKAGES=""
+ARG DOWNLOAD_COMMUNITY_PACKAGES=1
+ARG KOPANO_REPOSITORY_FLAGS="trusted=yes"
+ARG RELEASE_KEY_DOWNLOAD=0
+ARG DEBIAN_FRONTEND=noninteractive
+ARG KOPANO_CORE_REPOSITORY_URL="file:/kopano/repo/core"
+ARG KOPANO_CORE_VERSION=newest
+ARG KOPANO_KAPPS_REPOSITORY_URL="file:/kopano/repo/kapps"
+ARG KOPANO_KAPPS_VERSION=newest
+ENV KOPANO_KAPPS_VERSION=$KOPANO_KAPPS_VERSION
+
+ENV \
+    ADDITIONAL_KOPANO_PACKAGES=$ADDITIONAL_KOPANO_PACKAGES \
+    DOWNLOAD_COMMUNITY_PACKAGES=$DOWNLOAD_COMMUNITY_PACKAGES \
+    KOPANO_CORE_REPOSITORY_URL=$KOPANO_CORE_REPOSITORY_URL \
+    KOPANO_CORE_VERSION=$KOPANO_CORE_VERSION \
+    KOPANO_REPOSITORY_FLAGS=$KOPANO_REPOSITORY_FLAGS \
+    RELEASE_KEY_DOWNLOAD=$RELEASE_KEY_DOWNLOAD \
+    SERVICE_TO_START=kapps
```

Comment:
since this just starts a single service the variable could go (in the meet container as well)